server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -166,6 +167,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         final long previousWritesCompleted;
         final long currentSectorsWritten;
         final long previousSectorsWritten;
+        long currentIOTime;
+        long previousIOTime;
```

Comment:
Can we also mark these as `final`?

Reply:
Okay, I've changed it to final, thank you for check!